bin/gr_licenses

```diff
@@ -62,15 +83,29 @@ while [ $# -gt 0 ] ; do
   shift
 done
 
+license_finder_report="--format=$format --decisions_file=/policy/dependency_decisions.yml --columns=name version licenses approved license_links summary homepage"
+
 if [ -z "${projects}" ] ; then
   projects=${default_projects}
 fi
 
-function make_csv() {
-  local basename=${1%.txt}
-  sed -n -e '/<csv_start>/,/<csv_end>/p' "${basename}.txt" |\
-    sed -e '/^<csv.*>/d' -e '/^LicenseFinder::/d' |\
-    sort >"${basename}.csv"
+function make_csv_or_html() {
+  # Remove everything before and after result tags,
+  # Remove result tags and debug info,
+  # Sort if format is csv
+  sed -n -e '/<result>/,/<\/result>/p' "$1.txt" |\
+    sed -e '/^<result>/d' -e '/^<\/result>/d' -e '/^LicenseFinder::/d' |\
+    (test $format == "csv" && sort || cat) > "$1.$format"
+}
```

Comment:
Test question: Does this successfully filter out the lines that identify which package managers are being used?